virttest/utils_net.py

```diff
@@ -3744,3 +3746,190 @@ def map_hostname_ipaddress(hostname_ip_dict, session=None):
                 return False
     logging.info("All the hostnames and IPs are mapped in %s", hosts_file)
     return True
+
+
+def _remove_file(session, file_path):
```

Comment:
The implementation is windows only but hard to tell from the function name, also it is only used once, so I would recommend removing the function. You might have: `clean_cmd = "del %s"` and L#3897: `session_serial.cmd(clean_cmd % log_path)`, L#3922: `session_serial.cmd(clean_cmd % dump_file)`.